contributing-guide.md

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
Let's use this file name: `CONTRIBUTING.md`.

Reply:
Got it~ Done.